### PR TITLE
Add reminders with conditions 

### PR DESCRIPTION
**Pull request by MaxOSullivan**
Allows users to add reminders to conditions as the condition is being created

Screenshot 2022-09-05 at 15 24 09

Screenshot 2022-09-05 at 15 24 32

Screenshot 2022-09-05 at 15 24 54

### CODE REVIEW

File: client/components/conditions/add-conditions.js [hunk 1]
```diff
@@ -118,9 +121,10 @@ function AddConditions({
           onAdd={handleOnAdd}
           onUpdate={handleOnUpdate}
           onRemove={handleOnRemove}
+          conditionKey={'uuid'}
```
Comment (lennym) [inline client/components/conditions/add-conditions.js:124]: What does this do?

Comment (MaxOSullivan) [reply client/components/conditions/add-conditions.js:124]: This gives the custom condition a temporary condition key so a reminder can be saved against it in the state, then in the save function it is replaced when the UUID is actually created. Initially I tried to give it a real UUID at this point but it generated a new one every time it was rendered which broke things, so I opted for this way.

Comment (lennym) [reply client/components/conditions/add-conditions.js:124]: I think I see. The eventual uuid is generated at L142, but the downstream logic in the editable component wants a `conditionKey` to be set to allow reminders to be added. Is that right?

If so, could you just add a comment to the effect that the string being passed is an arbitrary placeholder and there's nothing special about the value `'uuid'`?

Comment (MaxOSullivan) [reply client/components/conditions/add-conditions.js:125]: Correct. Have added a comment above the line

**Pull request merged**